src/const/routes.ts

```diff
@@ -0,0 +1,9 @@
+import { TRoutes } from "../types/global"
```

Comment:
Nie musisz dla każdej zmiennej robić typu. Warto to robić gdy później chcesz ten typ wykorzystać w innych miejscach. Tutaj miejsca gdzie zaimportujesz routes będą znały jego typ, bo masz przypisane wartości (analogicznie jak pytałeś przy zmiennych).